library_generation/model/config_change.py

```diff
@@ -58,6 +58,35 @@ def __init__(self, changed_param: str, current_value: str, library_name: str = "
         self.library_name = library_name
 
 
+class NoOpConfigChange(LibraryChange):
```

Comment:
I couldn't find a source of truth on whether it makes sense to have an abstract class / interface if there is only one concrete class. I thought it could simplify the logic downstream to only deal with changes that should no-op instead of handling the case-specific non-monorepo change, mainly because it would future-proof it in case we want to add other no-op changes.

Reply:
That said, I think @blakeli0's comment may make these two classes unnecessary.